net/openssh/Portfile

```diff
@@ -129,27 +127,21 @@ if {${name} eq ${subport}} {
         # Old location(s):
         #   http://www.psc.edu/index.php/hpn-ssh
         # Current location(s):
-        #   http://hpnssh.sourceforge.net/
         #   http://www.freshports.org/security/openssh-portable/
         #     (is usually quick in updating the HPN patch for new versions,
         #      take a look there, too.)
 
-        # Formerly from FreeBSD, now copied over from FreeBSD's ports directory.
-        #patch_sites-append     http://mirror.shatow.net/freebsd/${name}/ \
-        #                       freebsd
-        #set hpn_patchfile      ${name}-6.7p1-hpnssh14v5.diff.gz
-        #checksums-append       ${hpn_patchfile} \
-        #                       rmd160  0cf7ffdd9b60d518d76076faf31df6a7a6d4ae52 \
-        #                       sha256  846ad51577de8308d60dbfaa58ba18d112d0732fdf21063ebc78407fc8e4a7b6
-
-        set hpn_patchfile       ${name}-${version}-hpnssh14v13.diff
-        patchfiles-append       ${hpn_patchfile}
+        # Source: FreeBSD /usr/ports/net/openssh-portable/files/extra-patch-hpn
+        # FreeBSD uses patch option `-p2', so first path prefix was removed
+        # from the original.
+        use_autoreconf          yes
+        patchfiles-append       ${name}-${version}-hpnssh14v15.diff
+        configure.args-append   --with-hpn --with-nonecipher
     }
 
     variant gsskex conflicts hpn requires kerberos5 description "Add OpenSSH GSSAPI key exchange patch" {
         use_autoreconf          yes
-        patchfiles-append       0002-Apple-keychain-integration-other-changes.patch \
-                                openssh-7.6p1-gsskex-all-20141021-mp-20171009.patch
+        patchfiles-append       gssapi.patch
```

Comment:
Thank you for efforts in integrating the HPN and GSSAPI patch! Could you add a link to the Debian GSSAPI patch? It will ease the next openssh update IMO.

Reply:
Done.  And since there doesn't appear to be wild opposition to temporarily dropping keychain support, that has been documented in the Portfile as well.